vignettes/invariants.md

```diff
@@ -282,10 +282,11 @@ exists.
 <td>
     tbl[[1:2]]
 
-    #> Warning: Calling `[[` with a vector
-    #> of length 2 (recursive subsetting)
-    #> is deprecated and will eventually be
-    #> converted to an error.
+    #> Warning: The `j` argument of
+    #> ``[[.tbl_df`()` can't be a vector of
+    #> length 2 as of lifecycle 3.0.0.
```

Comment:
Why does that have lifecycle instead of tibble?

Reply:
Good catch! I noticed this happens when the function is a method defined in another package. Easy to work around by adding `tibble::`, hard to reprex...